apps/firebase/rules.bolt

```diff
@@ -26,6 +26,14 @@ path /v3/channels/{channelId} {
   read() { canAccessChannel(channelId) }
 }
 
+path /v3/channels/shared {
+  read() { isLoggedIn() }
+}
+
+path /v3/channels/{channelId}/current_tables/{tableName} is Boolean {
+  write() { canAccessChannel(channelId) }
+}
```

Comment:
What does this do? i.e. It looks like it explicitly sets write permissions for tables that you own. Did we not have that before? If I don't own the channel and I'm using someone's app to fill out a form, will my response be recorded in their table?

Reply:
It sets write permissions on the `/current_tables/{tableName}` node for each channel. It also enforces that the value must be a boolean. I was following the pattern from the rest of the file to set the permission on the most specific node. I think this is because access is set hierarchically- access to a node also grants access to all of its descendants. `/current_tables/{tableName}` is a new descendant of `/v3/channels/{channelId}` that we're adding for the datasets work, so we need this rule to allow write access.

Not 100% sure about your second question- but I think the channel id is determined by the project itself, so if you're using someone's app, it access firebase through the channel id associated with *their* app. This change does not affect that, however.

Reply:
Makes sense. Thank you! 👍